paseos/actors/spacecraft_actor.py

```diff
@@ -23,7 +23,6 @@ def __init__(
         epoch: pk.epoch,
     ) -> None:
         """Constructor for a spacecraft actor
```

Comment:
Why are these white spaces removed? This does not comply with the [docstring format](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) we are using anymore